### PR TITLE
Logging hackathon - App.

### PR DESCRIPTION
**Pull request by M6AI**
## Bug / Requirement Description
Clearly and concisely describe the problem.

## Solution description
Describe your code changes in detail for reviewers.

## Checklist:
- [ ] Test
- [ ] Example (both test_plan.py and .rst)
- [ ] Documentation (API)
- [ ] News fragment present for release notes
- [ ] MS info leakage check
- [ ] For new driver: driver index page
- [ ] For new assertion: ui/pdf/std renderers, documentation
- [ ] For new cmdline arg: documentation

### CODE REVIEW

File: testplan/testing/multitest/driver/app.py [hunk 1]
```diff
@@ -454,10 +434,12 @@ def _move_std_and_logs(self):
...
                 archive(file, timestamp)
 
-    def aborting(self):
+    def aborting(self) -> None:
         """Abort logic to force kill the child binary."""
         if self.proc:
-            self.logger.debug("Killing process id {}".format(self.proc.pid))
+            self.logger.debug(
+                "Killing process id %s of App %s" % self.proc.pid, self.name
+            )
```
Comment (M6AI) [inline testplan/testing/multitest/driver/app.py:442]: Adding name to have non-process lookup as well.

File: testplan/testing/multitest/driver/app.py [hunk 2]
```diff
@@ -341,12 +323,11 @@ def starting(self):
                 self._proc = None
             raise
 
-    def started_check(self, timeout=None):
+    def started_check(self, timeout: Union[int, None] = None) -> None:
```
Comment (yuxuan-ms) [inline testplan/testing/multitest/driver/app.py:326]: Union[int, None] or Optional[int] ?

Comment (M6AI) [reply testplan/testing/multitest/driver/app.py:326]: Oh right, missed that return statement, updating.

File: testplan/testing/multitest/driver/app.py [hunk 3]
```diff
@@ -454,10 +432,12 @@ def _move_std_and_logs(self):
             if os.path.isfile(file):
                 archive(file, timestamp)
 
-    def aborting(self):
+    def aborting(self) -> None:
         """Abort logic to force kill the child binary."""
         if self.proc:
-            self.logger.debug("Killing process id {}".format(self.proc.pid))
+            self.logger.debug(
```
Comment (zhenyu-ms) [inline testplan/testing/multitest/driver/app.py:438]: I think it could be `info` here.

**Review by kn-ms — APPROVED** [review]

**Pull request merged**